packages/rest/test/resource.test.ts

```diff
@@ -163,14 +163,8 @@ describe("rest: resources", () => {
         @collectionAction(Thing, "export1")
         op exportThing(): {};
 
-        #suppress "deprecated"
         @post
         @collectionAction(Thing, "export2")
-        @segmentSeparator(":")
-        op exportThingWithColon1(): {};
```

Comment:
Looks like this operation was removed, is that intentional?

Reply:
Yes. This test existed to ensure segmentSeparator continued to work as before. Now that it is gone, we don't need it.